### PR TITLE
Coordinate conversion

### PR DESCRIPTION
**Pull request by bcail98**

### CODE REVIEW

File: client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js [hunk 1]
```diff
@@ -418,17 +433,31 @@ export default class CreateOrUpdateForm extends React.Component {
...
+                    const lonSec = parseFloat(lonValues[2]);
+
+                    //decimal = degrees + (minutes/60) + (seconds/3600)
+                    let lonDecimal = Math.abs(lonDegree) + (lonMin/60) + (lonSec/3600);
+                    if ((longitude.value.includes('W'))||(longitude.value.includes('w'))||(lonDegree<0)) {
+                        lonDecimal *= -1;
+                    }
+                    longitude.value = lonDecimal.toString();
+                }
+            }
```
Comment (renchauret) [inline client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js:455]: could be a function with lat or long passed in to avoid dupe code

Comment (bcail98) [reply client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js]: Done

**Comment by renchauret** [comment]
your change for my last comment looks good

played around with it a bit, couple things:
- could use some rounding, maybe to 6 decimal places. I got this off a conversion:  Coordinates: 43.08466666666667, -112.505916
- using anything other than decimal format does not trigger our address autofiller using Google Maps API.  So if you enter the same coordinates for two monuments, but in two different formats, one monument will get an address and the other will not.  Idk how complicated this is to fix.  If you think it's not too bad, add it to this, otherwise, please create a ticket for it on Trello

**Comment by bcail98** [comment]
> your change for my last comment looks good
> 
> played around with it a bit, couple things:
> 
> * could use some rounding, maybe to 6 decimal places. I got this off a conversion:  Coordinates: 43.08466666666667, -112.505916
> * using anything other than decimal format does not trigger our address autofiller using Google Maps API.  So if you enter the same coordinates for two monuments, but in two different formats, one monument will get an address and the other will not.  Idk how complicated this is to fix.  If you think it's not too bad, add it to this, otherwise, please create a ticket for it on Trello

Fixed both issues.

**Review by renchauret — APPROVED** [review]

**Pull request merged**